packages/ui/lib/components/icon/svg/IconWarning.tsx

```diff
@@ -4,7 +4,7 @@ const IconWarning = (props: SvgIconProps) => (
   <SvgIcon viewBox="0 0 48 48" {...props}>
     <path
       d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-      fill="currentColor"
+      fill="#F6E4C8"
```

Comment:
why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?

Reply:
In these three cases, the icons have a shadow that, when assigned with currentColor as happens a bit further below, results in the icon's shape not being displayed correctly.

![Screenshot 2024-12-09 alle 12 55 46](https://github.com/user-attachments/assets/d9f43912-3353-460e-a5f5-380b3c0acb78)


I haven't found an alternative at the moment.